src/frontend/web_application/src/components/ContactDetails/index.jsx

```diff
@@ -108,7 +124,7 @@ class ContactDetails extends Component {
           email={email}
           allowConnectRemoteEntity={allowConnectRemoteEntity}
           remoteIdentity={remoteIdentity}
-          editMode={this.state.editMode}
```

Comment:
I like to stick to full notation: `this.state.editMode` because it helps visually to distinguish what is props and what is state.

eg.

```
render() {
  const { foo } = this.props;

  return (<div><p>{foo}</p><span>({this.state.bar})</span></div>);
}
```